builtin/stash--helper.c

```diff
@@ -650,53 +650,67 @@ static int git_stash_config(const char *var, const char *value, void *cb)
 		show_patch = git_config_bool(var, value);
 		return 0;
 	}
+	return git_default_config(var, value, cb);
```

Comment:
Whoa. This hunk needs to be moved into the commit introducing `git_stash_config()`, as that commit would not even compile without a final `return`.